src/aat/resources/features/F-049/F-049.feature

```diff
@@ -0,0 +1,114 @@
+
+  @S-233 @Ignore
+  Scenario: must return 404 when request provide non-existing Idam user ID in CCD
+    Given   a user with [a detailed profile in CCD]
+    When    a request is prepared with appropriate values
+    And     the request [provide non-existing Idam user ID in CCD]
+    And     it is submitted to call the [Start case creation as Case worker] operation of [CCD Data Store]
+    Then    a negative response is received
+    And     the response [code is HTTP-403]
+    And     the response has all the details as expected
+
+  @S-233
+  Scenario: must return 403 when request provide non-existing Idam user ID in CCD
+    Given   a user with [a detailed profile in CCD]
+    When    a request is prepared with appropriate values
+    And     the request [provide non-existing Idam user ID in CCD]
+    And     it is submitted to call the [Start case creation as Case worker] operation of [CCD Data Store]
+    Then    a negative response is received
+    And     the response [code is HTTP-403]
+    And     the response has all the details as expected
+
+  @S-234 @Ignore
+  Scenario: must return 400 when request provide special character for Case type ID in CCD
+    Given   a user with [a detailed profile in CCD]
+    When    a request is prepared with appropriate values
+    And     the request [provide special character for Case type ID in CCD]
+    And     it is submitted to call the [Start case creation as Case worker] operation of [CCD Data Store]
+    Then    a negative response is received
+    And     the response [code is HTTP-400]
+    And     the response has all the details as expected
+
+  @S-511 @Ignore
+  Scenario: must return 404 when request provide a jurisdiction id not existing in ccd
+    Given   a user with [a detailed profile in CCD]
+    When    a request is prepared with appropriate values
+    And     the request [provide a jurisdiction id not existing in ccd]
+    And     it is submitted to call the [Start case creation as Case worker] operation of [CCD Data Store]
+    Then    a negative response is received
+    And     the response [code is HTTP-404]
+    And     the response has all the details as expected
+
+  @S-511
+  Scenario: must return 403 when request provide a jurisdiction id not existing in ccd
+    Given   a user with [a detailed profile in CCD]
+    When    a request is prepared with appropriate values
+    And     the request [provide a jurisdiction id not existing in ccd]
+    And     it is submitted to call the [Start case creation as Case worker] operation of [CCD Data Store]
+    Then    a negative response is received
+    And     the response [code is HTTP-403]
+    And     the response has all the details as expected
+
+  @S-512
+  Scenario: must return 404 when request provide an invalid Case type ID in CCD
+    Given   a user with [a detailed profile in CCD]
+    When    a request is prepared with appropriate values
+    And     the request [provide an invalid Case type ID in CCD]
```

Comment:
Grammar please please please

Reply:
as per your previous comment had changed i.e. 
please please use as proper a grammar as possible: here, this would require something like [provides an invalid Case Type ID]